rollup.config.js

```diff
@@ -3,14 +3,17 @@ import babel from '@rollup/plugin-babel';
 import commonjs from '@rollup/plugin-commonjs';
 import {terser} from 'rollup-plugin-terser';
 
-const config = mode => ({
+const config = ({mode, target}) => ({
   input: 'src/Recoil.js',
   output: {
-    file: `dist/recoil.${mode}.js`,
+    file:
+      target === 'web'
+        ? `dist/recoil.${mode}.js`
+        : `dist/recoil.${mode}.${target}.js`,
     format: 'cjs',
     exports: 'named',
   },
-  external: ['react', 'react-dom'],
+  external: ['react', 'react-dom', 'react-native'],
```

Comment:
`react-native` should probably not be resolved unless the target is `native`, likewise for `react-dom` and `browser`. (See the config in #108 for how to pull this off)

Reply:
This just ensures that the module isn't bundled with the library. It won't include the module if it isn't imported, so only including it when bundling for native won't change the output.